### DISCUSSION TITLE
Renaming tracks when disc is not recognized

### DISCUSSION THREAD

**Discussion by asherber** [enzo1982/freac#616] [Help & Support]
In CDex, if a disc I'm ripping is not recognized by CDDB, I can select the first track and hit F2 to enter a new name. When I hit Enter after typing the name, the focus moves to the next track, and I'm automatically in edit/rename mode. This makes it very easy to enter all the track names: F2, type, Enter, type, Enter, type, Enter, etc.

In freac, it looks like the only way to change a track name is to select it in the job list, then select the value in the Title field and enter the new name. Then I have to manually click on the next track, click back to the Title field, etc.

Am I missing something here? It would be nice to accomplish all of this just from the keyboard, so that I don't need to keep moving between keyboard and mouse. I don't see keyboard shortcuts for navigating around the window.

**Comment by jeenio** [comment]
Came here looking for an answer and 20 minutes later was writing an Autohotkeys entry for clicking the title field... Until, accidentally, found out that:

- after you typed the title for, say, track 1, just press ARROW DOWN

It would be nice to have the keyboard shortcuts, even if not editable, at least listed somewhere. Or maybe I don't know how to look them up.
